lib/ChatCommands/vehicle/repair.lua

```diff
@@ -1,24 +1,24 @@
--- Repair
-
-local cc_utils = require("chat_commander/utils")
-local vehicle_utils = require("chat_commander/vehicle_utils")
-
-return {
-    command="repair",
-    command_aliases={"fix"},
-    group="vehicle",
-    help="Repair any damage done to your current vehicle",
-    execute=function(pid, commands)
-        local vehicle = vehicle_utils.get_player_vehicle_in_control(pid)
-        if vehicle_utils.is_vehicle_command_ready(pid, vehicle) then
-            VEHICLE.SET_VEHICLE_FIXED(vehicle)
-            -- Also repair vehicle if its been destroyed by water
-            VEHICLE.SET_VEHICLE_DIRT_LEVEL(vehicle, 0.0)
-            VEHICLE.SET_VEHICLE_UNDRIVEABLE(vehicle, false)
-            VEHICLE.SET_VEHICLE_ENGINE_HEALTH(vehicle, 1000.0)
-            VEHICLE.SET_VEHICLE_PETROL_TANK_HEALTH(vehicle, 1000.0)
-            VEHICLE.SET_VEHICLE_ENGINE_ON(vehicle, true, true, true)
-            cc_utils.help_message(pid, "Vehicle repaired")
-        end
-    end
+-- Repair
+
+local cc_utils = require("chat_commander/utils")
+local vehicle_utils = require("chat_commander/vehicle_utils")
+
+return {
+    command="repair",
+    command_aliases={"fix"},
+    group="vehicle",
+    help="Repair any damage done to your current vehicle",
+    execute=function(pid, commands)
+        local vehicle = vehicle_utils.get_player_vehicle_in_control(pid)
+        if vehicle_utils.is_vehicle_command_ready(pid, vehicle) then
+            VEHICLE.SET_VEHICLE_FIXED(vehicle)
+            -- Also repair vehicle if its been destroyed by water
+            VEHICLE.SET_VEHICLE_DIRT_LEVEL(vehicle, 0.0)
+            VEHICLE.SET_VEHICLE_UNDRIVEABLE(vehicle, false)
+            VEHICLE.SET_VEHICLE_ENGINE_HEALTH(vehicle, 1000.0)
+            VEHICLE.SET_VEHICLE_PETROL_TANK_HEALTH(vehicle, 1000.0)
+            VEHICLE.SET_VEHICLE_ENGINE_ON(vehicle, true, true, true)
+            cc_utils.help_message(pid, "Vehicle repaired")
+        end
+    end
```

Comment:
anything changing on this file?

Reply:
nope

Reply:
> Sidenote: Im not sure why the !fix alias is still in this PR, even though there is no change to the file.

not sure why github thinks theres, changes, the files are 100% identical